shell/src/main/java/alluxio/cli/LogLevel.java

```diff
@@ -141,32 +154,68 @@ public static void logLevel(String[] args, AlluxioConfiguration alluxioConf)
         targets = new String[]{argTarget};
       }
     } else {
+      // By default we set on all targets (master/workers/job_master/job_workers)
       targets = new String[]{ROLE_MASTER, ROLE_JOB_MASTER, ROLE_WORKERS, ROLE_JOB_WORKERS};
     }
     return getTargetInfos(targets, conf);
   }
 
   private static List<TargetInfo> getTargetInfos(String[] targets, AlluxioConfiguration conf)
       throws IOException {
+    // Trim the elements
+    Set<String> targetSet = Arrays.stream(targets).map(String::trim).collect(Collectors.toSet());
     List<TargetInfo> targetInfoList = new ArrayList<>();
-    for (String target : targets) {
-      if (target.equals(ROLE_MASTER)) {
-        // TODO(jiacheng): make this support HA master
-        String masterHost = NetworkAddressUtils.getConnectHost(ServiceType.MASTER_WEB, conf);
+
+    // Allow plural form for the master/job_master and print a notice
+    if (targetSet.contains(ROLE_MASTERS) || targetSet.contains(ROLE_JOB_MASTERS)) {
+      System.out.println("The logLevel command will only take effect on the primary master, "
```

Comment:
primary master node may not be the primary job master node, can we give more clear warning message here for different roles?